bench-tps/src/bench.rs

```diff
@@ -413,6 +426,14 @@ fn poll_blockhash<T: BenchTpsClient>(
         let blockhash_updated = {
             let old_blockhash = *blockhash.read().unwrap();
             if let Some(new_blockhash) = get_new_latest_blockhash(client, &old_blockhash) {
+                datapoint_info!(
+                    "blockhash_stats",
+                    (
+                        "time_elapsed_since_blockhash_refreshed",
+                        blockhash_last_updated.elapsed().as_millis(),
+                        i64
+                    )
+                );
```

Comment:
isn't this one redundant with the `time_elapsed_since_last_blockhash_update` below?